Corvallis-Sustainability-Coalition/ReUse-Websites
# Render index with Slim.

- Adds sufficient session checking for admin sessions.
- Switches admin site to admin site 1 rather than 2.
- Misc. bug fixes and changes to admin panel.
- add session checking to all appropriate pages
- move all testing stuff to root/tests directory
- move/rename test helper to ApiTestCase in src directory
- make sure all tests inherit from base test case
- create example template render and fix search query
- donors render in php
- fix header function to act as proper form
- begin process of moving all site routes into Website.php
- handle drawing map pins using data attribute
- update template file so there is no way phpunit will try and run as test

Notes:  

- You'll have to do a `php composer.phar update`.  
- The new index implementation breaks the old one and the links on that page but I can't figure out a better way to do it then to progressively break the old site and move functionality to the newer / better version until its all completely implemented.

To Test:

1. Checkout my branch.
2. update composer
3. Run the dev server
4. View the index page
5. Run the tests.
6. Critique the code! (Notice... I added newlines!)
